### DISCUSSION TITLE
macOS shortcut to start VM from a list

### DISCUSSION THREAD

**Discussion by mjgardner** [utmapp/UTM#4224] [Show and tell]
Upon reading @ideologysec’s [comment](https://github.com/utmapp/UTM/issues/2280#issuecomment-1168026282) in issue #2280, I created the [macOS shortcut](https://support.apple.com/guide/shortcuts-mac) pictured below. It selects VMs from UTM’s container documents directory as well as any other folders you choose to add at the top of the shortcut. You can even choose multiple VMs to start in sequence.

You can download the shortcut from iCloud here: [Start UTM VM](https://www.icloud.com/shortcuts/0dfd3471cfa04d9c874d7ae8d952a233)

<img width="1281" alt="image" src="https://user-images.githubusercontent.com/133483/186787556-2c7fd200-449c-451d-8f1d-55bfd44f676a.png">

**Comment by ideologysec** [comment]
This is great, thank you. I'm working on an Alfred Workflow for this as well.

Are you able to share the shortcut itself, either via link or by publishing in the Shortcuts Gallery?

**Comment by ideologysec** [comment]
@mjgardner can you share this shortcut so people have an easy download / usability path?

**Reply by mjgardner** [reply]
Here you go, I’ve also edited the original post above. [Start UTM VM shortcut](https://www.icloud.com/shortcuts/0dfd3471cfa04d9c874d7ae8d952a233)

**Comment by ideologysec** [comment]
If anyone prefers Alfred Workflows, I've released one for controlling UTM virtual machines with Alfred:

https://github.com/ideologysec/alfred-utm-control
